tools/AzureRM.BootStrapper/AzureRM.BootStrapper.Tests.ps1

```diff
@@ -0,0 +1,231 @@
+Import-Module -Name AzureRM.Bootstrapper
```

Comment:
From Windows 10, pester modules are included in the OS install by default. Other OS's it can be installed by "Find-Module –Name 'Pester' | Install-Module". From the test folder, "Invoke-Pester" will run the tests. Should I include these instructions in a ReadMe.md or something?